go.mod

```diff
@@ -123,14 +123,15 @@ require (
 	github.com/hashicorp/golang-lru v1.0.2
 	github.com/hashicorp/hcl v1.0.1-vault-5
 	github.com/hashicorp/hcl/v2 v2.16.2
-	github.com/hashicorp/hcp-link v0.1.0
-	github.com/hashicorp/hcp-scada-provider v0.2.1
+	github.com/hashicorp/hcp-link v0.2.1
+	github.com/hashicorp/hcp-scada-provider v0.2.2
```

Comment:
I've hit many merge conflicts in this PR, because it's been open so long, and most of the conflicts are in `go.mod` and `go.sum`. Every time there's one of these conflicts, I manually edit `go.mod` to resolve the conflict, then run `go mod tidy` to re-generate the `go.sum` file. I think running `go mod tidy` is what's updating these.